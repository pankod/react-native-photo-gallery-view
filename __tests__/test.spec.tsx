describe('is test work?', () => {
	test('should return 4', () => {
		expect(2 + 2).toEqual(4);
		expect(2 + 2).toMatchSnapshot();
	});
});
