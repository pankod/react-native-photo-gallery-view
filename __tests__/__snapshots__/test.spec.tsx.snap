// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`is test work? should return 4 1`] = `4`;
